opentelemetry-api/src/opentelemetry/trace/__init__.py

```diff
@@ -191,6 +191,13 @@ def update_name(self, name: str) -> None:
         on the implementation.
         """
 
+    def is_recording_events(self) -> bool:
```

Comment:
You should return a boolean value (most likely `False`) here too. I wonder why mypy didn't catch this. 🤔

Reply:
mmm, so what about https://github.com/open-telemetry/opentelemetry-python/blob/be7577d6f2f2530433278e7393050ac6fe71b581/opentelemetry-api/src/opentelemetry/trace/__init__.py#L136?

Reply:
Seems also wrong.